### ISSUE TITLE
Incorrect flagging of JavaScript Object as 'Multiline Declaration'

### ISSUE THREAD

**Issue by Undistraction** [styled-components/stylelint-processor-styled-components#196]
This is OK:

```
const Layout = styled.div`
  ${example({ alpha: 1, bravo: { charlie: 3 } })};
`
```

This is flagged as an error:
```
const Layout = styled.div`
  ${example({ 
    alpha: 1, 
    bravo: { 
      charlie: 3,
    },
  })};
`
```

The latter is much more readable.

Error:

> [stylelint] Expected newline after ":" with a multi-line declaration (declaration-colon-newline-after)
> [stylelint] Expected no more than 0 empty lines (value-list-max-empty-lines)
> [stylelint] Expected no more than 0 empty lines (value-list-max-empty-lines)
> [stylelint] Expected no more than 0 empty lines (value-list-max-empty-lines)
> [stylelint] Expected no more than 0 empty lines (value-list-max-empty-lines)

![cap](https://user-images.githubusercontent.com/161489/40734621-df73fc7c-6430-11e8-9672-da11573950ae.jpg)

```
{
  "processors": ["stylelint-processor-styled-components"],
  "extends": [
    "stylelint-config-standard",
    "stylelint-config-styled-components"
  ],
  "syntax": "css",
  "rules": {
    "block-no-empty": null
  }
}
```

- stylelint-processor-styled-components@1.3.1
- stylelint@9.2.1

**Comment by Undistraction** [comment]
This is fixed by switching from `stylelint-config-default` to `stylelint-config-recommended`,
